agent/bench-scripts/pbench-fio

```diff
@@ -957,3 +985,7 @@ if [[ "$postprocess_only" == "n" ]]; then
 	fi
 	rmdir $benchmark_run_dir/.running
 fi
+
+# FYI this tricky using the exit status of `[[` itself, where we only exit with
+# zero if both statuses are 0, otherwise we exit with 1 indicating an error.
```

Comment:
Hmm; the sentence structure here is a bit weird. Maybe you meant "this is tricky" or "this is a trick"? (bash style trick or treat?) But probably not worth another commit to clarify since I assume any reader will likely figure it out.

Reply:
Fixed.